### PR TITLE
APP-1173 F/Unify package and plugin

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

- restore MVP functionalities
- prepare for split
- rename package to plugin
- rename Allowlist to addresslist

Task: [APP-1173](https://aragonassociation.atlassian.net/browse/APP-1173)

For QA : [deployed temporary subgraph](https://thegraph.com/hosted-service/subgraph/rekard0/rekard0-zaragoza-goerli) 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

**Comment by josemarinas** [comment]
Looks good, my only doubt is why are changes on the `permission-manager.test.ts`?

File: packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts [hunk 1]
```diff
@@ -47,12 +46,12 @@ let totalSupportThresholdPct = '500';
 let census = '1000';
 let actions = createDummyActions(DAO_TOKEN_ADDRESS, '0', '0x00000000');
 
-test('Run Allowlist Voting (handleVoteCreated) mappings with mock event', () => {
+test('Run Addresslist Voting (handleVoteCreated) mappings with mock event', () => {
   // create state
-  let erc20VotingPackage = new AllowlistPackage(
+  let erc20VotingPlugin = new AddresslistPlugin(
```
Comment (mathewmeconry) [inline packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts:51]: rename `erc20VotingPlugin` to `addresslistPlugin`

File: packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts [hunk 2]
```diff
@@ -229,17 +228,17 @@ test('Run Allowlist Voting (handleVoteExecuted) mappings with mock event', () =>
...
   clearStore();
 });
 
-test('Run Allowlist Voting (handleConfigUpdated) mappings with mock event', () => {
+test('Run Addresslist Voting (handleConfigUpdated) mappings with mock event', () => {
   // create state
   let entityID = Address.fromString(VOTING_ADDRESS).toHexString();
-  let erc20VotingPackage = new AllowlistPackage(entityID);
-  erc20VotingPackage.save();
+  let erc20VotingPlugin = new AddresslistPlugin(entityID);
```
Comment (mathewmeconry) [inline packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts:240]: rename `erc20VotingPlugin` to `addresslistPlugin`

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts [hunk 3]
```diff
@@ -173,24 +194,24 @@ test('UpdatePrepared event (non existent plugin)', function() {
...
-  assert.fieldEquals(
-    'Plugin',
-    pluginId,
-    'dao',
-    Address.fromHexString(DAO_ADDRESS).toHexString()
-  );
-  assert.fieldEquals('Plugin', pluginId, 'pluginSetup', ADDRESS_TWO);
-  assert.fieldEquals('Plugin', pluginId, 'state', 'UpdatePrepared');
+  assert.notInStore('Plugin', pluginId);
+  // assert.fieldEquals(
```
Comment (mathewmeconry) [inline packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:198]: remove unnecessary code

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts [hunk 4]
```diff
@@ -230,14 +251,14 @@ test('UpdateApplied event (non existent plugin)', function() {
...
-  assert.fieldEquals('Plugin', pluginId, 'sender', ADDRESS_ZERO);
-  assert.fieldEquals(
-    'Plugin',
-    pluginId,
-    'dao',
-    Address.fromHexString(DAO_ADDRESS).toHexString()
-  );
-  assert.fieldEquals('Plugin', pluginId, 'state', 'Installed');
+  assert.notInStore('Plugin', pluginId);
+  // assert.fieldEquals(
```
Comment (mathewmeconry) [inline packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:255]: remove unnecessary code

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts [hunk 5]
```diff
@@ -302,24 +323,24 @@ test('UninstallationPrepared event (non existent plugin)', function() {
...
-  assert.fieldEquals(
-    'Plugin',
-    pluginId,
-    'dao',
-    Address.fromHexString(DAO_ADDRESS).toHexString()
-  );
-  assert.fieldEquals('Plugin', pluginId, 'pluginSetup', ADDRESS_TWO);
-  assert.fieldEquals('Plugin', pluginId, 'state', 'UninstallPrepared');
+  assert.notInStore('Plugin', pluginId);
+  // assert.fieldEquals(
```
Comment (mathewmeconry) [inline packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:327]: remove unnecessary code

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts [hunk 6]
```diff
@@ -359,14 +380,14 @@ test('UninstallationApplied event (non existent plugin)', function() {
...
-  assert.fieldEquals('Plugin', pluginId, 'sender', ADDRESS_ZERO);
-  assert.fieldEquals(
-    'Plugin',
-    pluginId,
-    'dao',
-    Address.fromHexString(DAO_ADDRESS).toHexString()
-  );
-  assert.fieldEquals('Plugin', pluginId, 'state', 'Uninstalled');
+  assert.notInStore('Plugin', pluginId);
+  // assert.fieldEquals(
```
Comment (mathewmeconry) [inline packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:384]: remove unnecessary code

**Review by mathewmeconry — CHANGES_REQUESTED** [review]

File: packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts (hunk 1 above)
Comment (Rekard0) [reply packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts:51]: Amended

File: packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts (hunk 2 above)
Comment (Rekard0) [reply packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts:240]: Amended

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts (hunk 3 above)
Comment (Rekard0) [reply packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:198]: Amended

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts (hunk 4 above)
Comment (Rekard0) [reply packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:255]: Amended

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts (hunk 5 above)
Comment (Rekard0) [reply packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:327]: Amended

File: packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts (hunk 6 above)
Comment (Rekard0) [reply packages/subgraph/tests/plugin/pluginSetupProcessor.test.ts:384]: Amended

**Comment by Rekard0** [comment]
> Looks good, my only doubt is why are changes on the `permission-manager.test.ts`?

**Review by mathewmeconry — APPROVED** [review]

**Review by josemarinas — APPROVED** [review]

**Pull request merged**